Refit/VV-Refit.csproj

```diff
@@ -2,7 +2,9 @@
 
```

Comment:
This project should probably not be renamed.

Reply:
Due some mistakes, I made another pull request with all fixed. #414